src/app/zap-templates/zcl/data-model/chip/media-playback-cluster.xml

```diff
@@ -161,4 +161,18 @@ limitations under the License.
     <item name="SpeedOutOfRange" value="0x04"/>
     <item name="SeekOutOfRange" value="0x05"/>
   </enum>
+
+  <enum name="MediaPlaybackRequest" type="ENUM8">
```

Comment:
I don't see this enum in the spec.  Where is it coming from?

Reply:
I've needed a data structure to share between the clusters and example app code. I've removed this enum and I've created a media-playback-server.h where I've declared this.